market-service/src/main/java/rs/edu/raf/banka1/model/Listing.java

```diff
@@ -16,7 +15,7 @@
 @Getter
 @Setter
 @NoArgsConstructor
-public class ListingModel {
```

Comment:
Baci pogled na endpoints fajl i vidi da li se slazu tipovi/imena/ime klase

Reply:
Nije više validno, pošto smo odradili refactor koda sa onim nasljeđivanjem.